contracts/FlowIDTableStaking.cdc

```diff
@@ -300,22 +280,11 @@ pub contract FlowIDTableStaking {
     /// Records the staking info associated with a delegator
     /// Stored in the node's NodeRecord
     pub resource DelegatorRecord {
-        /// Tokens this delegator has committed for the next epoch
```

Comment:
Any reason the documentation comments were removed?

Reply:
So I could fit the contract in the max transaction size. I'd like to have them in there, but I can't fit them unfortunately

Reply:
@ramtinms Do we have a way for the service account to send larger transaction then the default?